tracer/build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -157,6 +157,29 @@ public void Run()
                 "src/Datadog.Trace.ClrProfiler.Native/version.h",
                 text => FullVersionReplace(text, "."));
 
+            // .NET profiler
+
+            SynchronizeVersion(
+                "../profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/Resource.rc",
+                text =>
+                {
+                    text = FullVersionReplace(text, ",");
+                    text = FullVersionReplace(text, ".");
+                    return text;
+                });
+
+            SynchronizeVersion(
```

Comment:
Did you run this after adding the Nuke target? I would have expected the version in the CMakeLists.txt to be updated in this PR as well

Reply:
I do not know if the version update should be executing along (it seems not)
I ran it locally and the CMakeLists.txt file was modified.
Do you mean I should update the version in the CMakeLists.txt in this PR? (FYI, we do not ship the linux version yet)

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2530/commits/ce176017c5ca872f87c0be6c8f784c4b0828be85

Reply:
> Do you mean I should run the update the version target and push all the modifications in this PR
Yep, exactly, now everything looks correct, thanks 🙂

Reply:
Perfect, thanks!